snapcraft/parts/lifecycle.py

```diff
@@ -545,6 +545,7 @@ def _run_in_provider(
         base_configuration=base_configuration,
         build_base=build_base.value,
         instance_name=instance_name,
+        allow_unstable=True,  # type: ignore  # TODO: fix in craft-providers 1.8.2
```

Comment:
Is it ok to always set `allow_unstable` or should I check the base before setting this flag?

Reply:
I would check

Reply:
Done!  

I also added a warning that appears for all verbosity levels except `quiet`.